conn.go

```diff
@@ -41,13 +47,24 @@ func NewConn(conn io.ReadWriteCloser) *Conn {
 	return c
 }
 
-// Dial connects to the given address on the given network using net.Dial
-// and then returns a new Conn for the connection.
+// Dial connects to the given address on the given network using net.DialTimeout
+// with a default timeout of 10s and then returns a new Conn for the connection.
 func Dial(network, addr string) (*Conn, error) {
-	c, err := net.Dial(network, addr)
+	connTimeout := time.Duration(DefaultConnTimeout) * time.Second
+	return DialTimeout(network, addr, connTimeout)
```

Comment:
And if you like the above suggestions, then this would of course be:

```suggestion
	return DialTimeout(network, addr, DefaultConnTimeout)
```